container_pipeline/workers/build.py

```diff
@@ -87,7 +87,9 @@ def handle_build_success(self, job):
 
     def handle_build_failure(self, job):
         """Handle build failure for job"""
-        self.queue.put(json.dumps(job))
+        job.pop('action', None)
+        job['action'] = "build_failure"
```

Comment:
What I think is, the operations in two lines can be clubbed into one line only and it is not giving risk of repeating emails.